s3-cloudfront-spa/example-pattern.json

```diff
@@ -0,0 +1,63 @@
+{
+  "title": "AWS CloudFront and Amazon S3 for Single Page Application Hosting",
+  "description": "This project shows how to host a Single-Page Application (SPA) using AWS CloudFront & S3. CloudFront serves the SPA content from an S3 bucket configured for static hosting.",
+  "level": "200",
+  "framework": "Terraform",
```

Comment:
Please also add the language as a top level attribute

Reply:
language added.